docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.fonts.scss

```diff
@@ -0,0 +1,126 @@
+// Started using humsci_basic `hb` prefix here
+// Thoughts?
+@mixin hb_heading-1 {
+  @include colorful {
+    font-weight: theme-font-weight(semibold);
+    font-size: 40px;
+    line-height: 126%;
+
+    @include grid-media-min("md") {
+      font-size: 58px;
+    }
+  }
+}
+
+@mixin hb_heading-2 {
+  @include colorful {
+    font-weight: theme-font-weight(semibold);
+    font-size: 33px;
+    line-height: 125%;
+
+    @include grid-media-min("md") {
+      font-size: 48px;
+    }
+  }
+}
+
+@mixin hb_heading-3 {
+  @include colorful {
+    font-weight: theme-font-weight(semibold);
+    font-size: 27px;
+    line-height: 109%;
+
+    @include grid-media-min("md") {
+      font-size: 38px;
+    }
+  }
+}
+
+@mixin hb_heading-4 {
+  @include colorful {
+    font-weight: theme-font-weight(semibold);
+    font-size: 22px;
+    line-height: 121%;
+
+    @include grid-media-min("md") {
+      font-size: 32px;
+    }
+  }
+}
+
+@mixin hb_heading-5 {
+  @include colorful {
+    font-weight: theme-font-weight(regular);
+    font-size: 22px;
+    line-height: 117%;
+  }
+}
+
+@mixin hb_heading-6 {
+  @include colorful {
+    text-transform: uppercase;
+    font-weight: theme-font-weight(semibold);
+    font-size: 16px;
+    line-height: 117%;
+  }
+}
+
+@mixin hb_subheading {
+  @include colorful {
+    font-weight: theme-font-weight(semibold);
+    font-size: 15px;
+    line-height: 131%;
+  }
+}
+
+@mixin hb_body--small {
+  @include colorful {
+    font-weight: theme-font-weight(regular);
+    font-size: 16px;
+    line-height: 122%;
+  }
+}
+
+@mixin hb_body--medium {
+  @include colorful {
+    font-weight: theme-font-weight(regular);
+    font-size: 16px;
+    line-height: 123%;
+
+    @include grid-media-min("md") {
+      font-size: 18px;
+      line-height: 127%;
+    }
+  }
+}
+
+@mixin hb_link {
+  // no font size by default
+  @include colorful {
+    color: darken($su-color-driftwood, 45%);
+    text-decoration: none;
+    font-weight: theme-font-weight(bold);
+    margin-bottom: 3px;
+
+    &:hover,
+    &:focus {
+      box-shadow: inset 0 -9px 0 lighten(colorful-theme--variation(primary), 70%);
+      border-bottom: 2px solid lighten(colorful-theme--variation(primary), 20%);
+      // accommodate for 2px border bottom by reducing margin
+      margin-bottom: 1px;
```

Comment:
Side note: when updating rems you cannot convert 1px to rem.

Reply:
really?! that's so weird. can you send me an article about this?

Reply:
Ignore me. When rems first came out I had trouble converting 1px, it just didn't work, but after playing around with it tonight that's not the case any more 🤦‍♀

Reply:
ha nooo worries :)